wardenclyffe/main/views.py

```diff
@@ -550,21 +529,10 @@ def s3_batch_upload(request, collection_id):
             language=request.POST.get('language_' + idx, ""),
         )
 
-        source_file = v.make_source_file(key)
-        label = "uploaded source file (S3)"
-        source_file = File.objects.create(video=v, url="", cap=key,
-                                          location_type="s3",
-                                          filename=key,
-                                          label=label)
-
-        v_operations = [
-            v.make_pull_from_s3_and_extract_metadata_operation(
-                key=key, user=request.user),
-            v.make_create_elastic_transcoder_job_operation(
-                key=key, user=request.user)]
-        if v.collection.audio:
-            o = v.make_audio_encode_operation(source_file.id, request.user)
-            v_operations.append(o)
+        v.make_source_file(key)
+        v.make_uploaded_source_file(key)
+        v_operations = v.initial_operations(key, request.user,
```

Comment:
The initial_operations function aligns with the existing code at [lines 496-505](https://github.com/ccnmtl/wardenclyffe/blob/master/wardenclyffe/main/views.py#L497). The refactored call at [line 483](https://github.com/ccnmtl/wardenclyffe/blob/0d5d4afd966b0179e22d18a65e2621539c055588/wardenclyffe/main/views.py#L483) seems correct.

However, the call to initial_operations here doesn't seem to match up with the existing code at [lines 560-69](https://github.com/ccnmtl/wardenclyffe/blob/master/wardenclyffe/main/views.py#L560) where a make_pull_from_s3_and_extract_metadata_operation and a make_create_elastic_transcoder_job_operation are completed in addition to a make_audio_encode_operation if audio.